### PR TITLE
disabling sampling of requests in default host.json

### PR DESCRIPTION
**Pull request by soninaren**
closes https://github.com/Azure/azure-functions-templates/issues/906

### CODE REVIEW

File: src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs [hunk 1]
```diff
@@ -434,5 +436,15 @@ private static async Task<string> AddBundleConfig(string hostJsonContent)
             hostJsonObj.Add("extensionBundle", bundleConfig);
             return JsonConvert.SerializeObject(hostJsonObj, Formatting.Indented);
         }
+
+
+        private static async Task<string> AddLoggingConfig(string hostJsonContent)
```
Comment (ahmelsayed) [inline src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs]: any reason for this approach as opposed to just have it in the default host.json?

Comment (soninaren) [reply src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs]: I followed the sample of setting bundles as default in host.json. I think the only advantage we get by following this approach, is that we can introduce a flag to skip this logic in the init command `no-loggingConfig`. While that made sense for bundles, I don't think it does for Application Insight configuration. I will revert this change to simple add this configuration to default host.json

Comment (ahmelsayed) [reply src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs]: sounds good

**Comment by brettsam** [comment]
While you are doing this -- @jeffhollan (or @paulbatum... I can't remember who brought it up), weren't we discussing also adding this to the default?

```
"samplingSettings": {
    "isEnabled": true 
}
```

So it'd be much more "in your face" that sampling is enabled by default?

**Comment by soninaren** [comment]
> While you are doing this -- @jeffhollan (or @paulbatum... I can't remember who brought it up), weren't we discussing also adding this to the default?
> 
> ```
> "samplingSettings": {
>     "isEnabled": true 
> }
> ```
> 
> So it'd be much more "in your face" that sampling is enabled by default?

Jeff added that to the issue after the PR was out. Will add that to the config as well

**Pull request merged**